### PR TITLE
Adding dataprep

### PR DESCRIPTION
**Pull request by dovahcrow**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dovahcrow** [comment]
@conda-forge-admin Hi team, this PR is ready for review.

File: recipes/dataprep/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+requirements:
+  build:
+    - pip
+    - python >=3.6
+    - poetry >=1
+  run:
+    - python >=3.6
+    - bokeh >=2.0,<2.1
+    - dask[complete] >=2.13,<2.14
```
Comment (scopatz) [inline recipes/dataprep/meta.yaml:25]: conda does not recognize features like this. Please list all dependencies separately

File: recipes/dataprep/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - jsonpath-ng >=1.5.1,<2.0.0
+    - jsonschema >=3.2,<3.3
+    - lxml >=4.5,<4.6
+    - nltk >=3.5,<4.0
+    - numpy >=1.18,<1.19
+    - pandas >=1.0,<1.1
+    - pillow >=7.1.2,<8.0.0
+    - requests >=2.23,<2.24
+    - scipy >=1.4,<1.5
+    - tornado ==5.0.2
```
Comment (scopatz) [inline recipes/dataprep/meta.yaml:37]: Does this really need to be pinned so tightly?

Comment (dovahcrow) [reply recipes/dataprep/meta.yaml:37]: Yep, this is due to https://github.com/sfu-db/dataprep/issues/218.

File: recipes/dataprep/meta.yaml (hunk 1 above)
Comment (dovahcrow) [reply recipes/dataprep/meta.yaml]: @scopatz fixed. BTW do you know if there's any tool I can directly convert pypi requirements to conda format? What I did is basically copy/past/dedupe

Comment (scopatz) [reply recipes/dataprep/meta.yaml]: Nope, there isn't one yet

File: recipes/dataprep/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{% set name = "dataprep" %}
+{% set version = "0.2.10a0" %}
```
Comment (scopatz) [inline recipes/dataprep/meta.yaml:2]: Please don't include `a0` in the version number.

Comment (dovahcrow) [reply recipes/dataprep/meta.yaml:2]: We haven't released version 0.2.10 yet. Can we use a0 here and I'll create a new PR for the feedstock after we release 0.2.10 by the end of this week?

Comment (scopatz) [reply recipes/dataprep/meta.yaml:2]: We don't release alpha or dev versions without a special mechanism.  https://github.com/conda-forge/cfep/blob/ac7d56c5efbc0f9b2f45130a658103b24506736a/cfep-05.md

It is probably easier to just wait

Comment (dovahcrow) [reply recipes/dataprep/meta.yaml:2]: Agreed. Will be back at the end of the week.

Comment (scopatz) [reply recipes/dataprep/meta.yaml:2]: Sounds great!

**Comment by dovahcrow** [comment]
Hi @scopatz We released v0.2.10. This recipe is ready to review again.

**Comment by scopatz** [comment]
Thanks!

**Comment by dovahcrow** [comment]
Hi @scopatz, do you know if I can use anaconda 5.x to build the recipe? The CI in this PR passed but it failed to build the recipe for the feedstock https://github.com/conda-forge/dataprep-feedstock. I tested locally and found the docker image used to build the recipe is on anaconda 4.8.3. I think there are some package resolving issues for anaconda 4.x.

**Pull request merged**